pgxn/neon/control_plane_connector.c

```diff
@@ -802,6 +803,14 @@ NeonProcessUtility(
 		case T_DropRoleStmt:
 			HandleDropRole(castNode(DropRoleStmt, parseTree));
 			break;
+		case T_CreateTableSpaceStmt:
+			if (!RegressTestMode)
+			{
+				ereport(ERROR,
+					(errcode(ERRCODE_INSUFFICIENT_PRIVILEGE),
+					errmsg("CREATE TABLESPACE statements are not supported by Neon.")));
```

Comment:
just to follow the usual postgres message style more closely: 
```suggestion
					errmsg("CREATE TABLESPACE is not supported on Neon")));
```